plugins/woocommerce/src/Blocks/BlockTypes/ProductDetails.php

```diff
@@ -1,15 +1,19 @@
-<?php declare( strict_types = 1 );
+<?php
+
+declare(strict_types=1);
 
 namespace Automattic\WooCommerce\Blocks\BlockTypes;
 
 use WP_Block;
 use WP_HTML_Tag_Processor;
 use Automattic\WooCommerce\Blocks\Utils\StyleAttributesUtils;
+use Automattic\WooCommerce\Blocks\Utils\Utils;
 
 /**
  * ProductDetails class.
  */
-class ProductDetails extends AbstractBlock {
+class ProductDetails extends AbstractBlock
+{
```

Comment:
We need to keep this formatting as it was, otherwise Lint automations are going to fail.